### ISSUE TITLE
I can't get my AMD modules loaded for testing?

### ISSUE THREAD

**Issue by Integralist** [cloudchen/grunt-template-jasmine-requirejs#15]
Hi,

I've set-up a [Grunt boilerplate](https://github.com/Integralist/Grunt-Boilerplate) where you can see all my files (although I'll include some code below for reference).

I've got Jasmine running via Grunt no problem. But when I try and get Jasmine to test my AMD modules it fails to load the modules, so the tests fail.

You'll see my Grunt file contains the following snippet...

``` js
jasmine: {
    src: ['app/**/*.js', '!app/release/**'],
    options: {
        specs: 'specs/**/*Spec.js',
        helpers: 'specs/helpers/*Helper.js',
        template: require('grunt-template-jasmine-requirejs'),
        templateOptions: {
            requireConfig: {
                baseUrl: '.app/',
                mainConfigFile: './app/main.js'
            }
        }
    }
}
```

This successfully loads both my SpecHelper.js file and my BasicSpec.js file.

BasicSpec.js consists of...

``` js
describe('Basic', function(){
    it('should pass a single assertion without causing an error', function(){
        expect([]).toBeArray(); // this matcher is custom set within my SpecHelper.js file
    })

    it('should let me test our AMD modules', function(){
        expect(window.xyz).toBe(123); // I set this global property within my /app/main.js file require callback
    })
});
```

If we look at my app/main.js file we'll see...

``` js
require.config({ 
    paths: {
        a: '../modules/a',
        b: '../modules/b',
        c: '../modules/c',
        jquery: '../lib/jquery'
    }
});

require(['a', 'b', 'c', 'jquery'], function (a, b, c, $) {
    console.log(a);
    console.log(b);
    console.log(c);
    console.log($);
    window.xyz = 123;
});
```

You can see I set a global property (which I did at first to see if my main.js and the required modules were even being loaded) and I try and test that value in my BasicSpec.js file but that test fails.

I'm also not sure how I would go about testing my modules `a`, `b`, and `c` because they're inside a closure? Hence why I started with setting a global property to see if I could at least access that.

On a side note: I noticed that a `_SpecRunner.html` file was being generated all of a sudden (it wasn't at first so I've no idea how I managed to get that file generated - or the `.grunt` directory that came along with it?) -> looking at that HTML file I can see it's picking up my config settings for RequireJS but I can see it never attempts to load the modules within my main application js file. 

In short: how can I get this to work? (please refer to my repo linked above for full source code)

Apologies if I'm just being very stupid, but I've looked into this for a while now and I've still not managed to wrap my head around it or found a solution.

Thanks.
